pkg/validation/exception/validate.go

```diff
@@ -7,8 +7,24 @@ import (
 	kyvernov2alpha1 "github.com/kyverno/kyverno/api/kyverno/v2alpha1"
 )
 
+const (
+	namespacesDontMatch = "PolicyException resource namespace must match the defined namespace."
+	disabledPolex       = "PolicyException resources would not be processed until it is enabled."
+)
+
+type ValidationOptions struct {
+	EnablePolex    bool
+	PolexNamespace string
+}
+
 // Validate checks policy exception is valid
-func Validate(ctx context.Context, logger logr.Logger, polex *kyvernov2alpha1.PolicyException) error {
+func Validate(ctx context.Context, logger logr.Logger, polex *kyvernov2alpha1.PolicyException, opts ValidationOptions) ([]string, error) {
```

Comment:
@yimikao can we add a couple unit tests here ?

Reply:
sure

Reply:
@eddycharly done